lib/mixins/dropdown.js

```diff
@@ -80,7 +81,7 @@ export default {
             }
         }
     },
-    methods: {
+    methods: Object.assign({}, clickOut.methods, {
         noop() {
```

Comment:
I wasn't sure if we could use spread here or not since IE sucks at it without a transpiler.

![image](https://user-images.githubusercontent.com/2781561/27810064-438f34ce-602c-11e7-93ac-e60e9f5f7993.png)